dapps/marketplace/src/pages/_Footer.js

```diff
@@ -74,25 +74,29 @@ class Footer extends Component {
                       </div>
                       <br />
                       <div>
-                        {'Found a bug or have feedback? Send an email to '}
-                        <a href={`mailto:${SupportEmail}`}>{SupportEmail}</a>
-                        {', open an issue on '}
-                        <a
-                          href={GitHubLink}
-                          target="_blank"
-                          rel="noopener noreferrer"
-                        >
-                          GitHub
-                        </a>
-                        {' or post in our #bug-reports channel on '}
-                        <a
-                          href="https://discord.gg/jyxpUSe"
-                          target="_blank"
-                          rel="noopener noreferrer"
-                        >
-                          Discord
-                        </a>
-                        .
+                        <fbt desc="footer.foundABug">
+                          Found a bug or have feedback? Send an email to
+                          <a href={`mailto:${SupportEmail}`}>
+                            <fbt:param name="SupportEmail">
+                              {SupportEmail}
+                            </fbt:param>
+                          </a>, open an issue on
```

Comment:
Does this end up correctly putting a space between "on" and the link ? Same question on line #91 below.

Reply:
Spacing looks right to me: 

![image](https://user-images.githubusercontent.com/673455/55517182-d7c3e480-562c-11e9-87f6-b7c20133393a.png)

![image](https://user-images.githubusercontent.com/673455/55517207-e7432d80-562c-11e9-958f-be20af18c0f6.png)